CHANGES.md

```diff
@@ -1,6 +1,162 @@
 Release Changes List
 ====================
 
+version 2.0
+===========
+
+released March 21, 2024
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+
+Syntactic / Naming Changes
+--------------------------
+
+Language Feature Improvements
+-----------------------------
+
+Semantic Changes / Changes to the Chapel Language
+-------------------------------------------------
+
+Deprecated / Unstable / Removed Language Features
+-------------------------------------------------
+
+Namespace Changes
+-----------------
+
+Standard Library Modules
+------------------------
+
+Package Modules
+---------------
+
+Standard Domain Maps (Layouts and Distributions)
+------------------------------------------------
+
+Changes / Feature Improvements in Libraries
+-------------------------------------------
+
+Name Changes in Libraries
+-------------------------
+
+Package Modules
```

Comment:
just noting Package Modules is in here twice

Reply:
:o